roles/ovirt-datacenter-cleanup/tasks/vms.yml

```diff
@@ -0,0 +1,13 @@
+- name: Find existing VMs
+  ovirt_vms_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }}"
+
+- name: Cleanup VMs
```

Comment:
I would change Cleanup to Remove in most roles

Reply:
Done.